### PR TITLE
mx_player

### PR DESCRIPTION
**Pull request by abhijith041**
DDG Community Kochi
#2127

### CODE REVIEW

**Comment by daxtheduck** [comment]
@abhijith041 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/mx_player.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
+{
+  "id": "mxplayer_cheat_sheet",
+  "name": "mx player",
+  "description": "Keyboard shortcuts for Brackets, an open source text editor from mx  player",
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/json/mx_player.json]: @abhijith041 Is this a Cheat Sheet about Brackets or Mx Player?

**Comment by MariagraziaAlastra** [comment]
Hey @abhijith041, thanks for your contribution, I just left a comment in the code.
Could you please create an IA Page for your Cheat Sheet [here](https://duck.co/ia/new_ia) and link to it in your first comment? 
You can find more info [in the docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request)

**Comment by moollaza** [comment]
>Could you please create an IA Page for your Cheat Sheet here and link to it in your first comment?
You can find more info in the docs

@abhijith041 once you've made an IA page, the ID and Filename will also need to match the ID of the IA page.

I.e.
- IA Page ID: mx_player_cheat_sheet
- Cheat Sheet `"id": "mx_player_cheat_sheet"
- Filename: "mx-player.json"

**Comment by alohaas** [comment]
@abhijith041, thanks again for creating the mx player cheat sheet! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge – our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by mintsoft** [comment]
@abhijith041 I'm going to close this PR; feel free to reopen it whenever you like :+1:

**Pull request closed without merging**